libultraship/libultraship/Lib/Fast3D/gfx_glx.cpp

```diff
@@ -276,8 +276,8 @@ static void gfx_glx_set_fullscreen_state(bool on, bool call_callback) {
     xev.xclient.data.l[2] = 0;
     xev.xclient.data.l[3] = 0;
     XSendEvent(glx.dpy, glx.root, 0, SubstructureNotifyMask | SubstructureRedirectMask, &xev);
-    gfx_glx_ShowHideMouse(on);
-    
+    gfx_glx_show_cursor(on);
```

Comment:
Is this a refactor for the sake of it? Or was this a mistake in the original code?

Reply:
This is a mistake in the original code. Someone changed the name of this function and didn't apply the change everywhere needed.